app/views/games/show.html.erb

```diff
@@ -7,15 +5,13 @@
 				<% (0..7).each do |row| %>
 					<tr>
 					<% (0..7).each do |col|  %>
-						<% if ((row % 2 == 0) && (col % 2 != 0)) || ((row % 2 != 0) && (col % 2 == 0)) %>
-							<td class="wht">
-								<a href="#"><%= col %></a>
+							<td class="<%= color_block(row, col)%>">
+								<% @pieces.each do |piece| %>
+									<% if (piece.row_position == row) && (piece.col_position == col) %> 
+										<%= link_to "#{piece.type} #{piece.color}" %>
```

Comment:
Good placeholder.  Feel free to start a discussion or set up pair-programming with your team about what this link will need to do.